index.html

```diff
@@ -18,10 +18,18 @@
             <div class="logo_nav">
                 <a href="#"><img class="image_nav" src="images/my_logo.svg" alt="logo"></a>
             </div>
-            <a href="#"><img class="menu_icon" src="images/menu.svg" alt="menu"></a>
+            <div class="nav_content">
+                <a href="#"><img class="menu_icon" src="images/menu.svg" alt="menu"></a>
+                <ul class="nav_menu">
+                    <li><a class="nav_li_item" href="#portfolio">Portfolio</a></li>
+                    <li><a class="nav_li_item" href="#about_me">About me</a></li>
+                    <li><a class="nav_li_item" href="#contact">Contact</a></li>
+                </ul>
+            </div>
         </nav>
         <section class="first_section">
-            <h1 class="title content">I’m Alvaro. Glad to see you!</h1>
+            <div class="content_header">
+            <h1 class="title content">I’m Alvaro. <br> Glad to see you!</h1>
             <p class="paragraph content">I’m a software developer! I can help you build a product , feature or website
                 Look
                 through some of my work and experience! If you like what you see and have a project you need coded,
```

Comment:
Please consider using the same styles for the paragraph as being used in the design like `letter spacing, font-size, etc` to make it look exactly like the design.

## Solution:-

Remember, to copy the styles of an element click on it in Figma and copy the styles from the right panel or side bar.